File: ps6/sentimental-mario-more/mario.py
from cs50 import get_int
height = 0
while height < 1 or height > 8:
    height = get_int("Height: ")

for w in range(height):
    if w < height:
        print(' ' * ((height-1)-w) + '#' * (w+1) + ' ' * 2 + '#' * (w+1))

File: cs50x/ps1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void print_row(int length, int spaces);

int main()
{
    // Get height 1-8
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height > 8 || height < 1);
    int sub = height - 1;
    for (int i = 0; i < height; i++)
    {
        print_row(i + 1, sub);
        sub--;
    }
}

// Print Rows
void print_row(int length, int spaces)
{
    for (; spaces > 0; spaces--)
    {
        printf(" ");
    }

    for (int i = 0; i < length; i++)
    {
        printf("#");
    }
    printf("  ");
    for (int i = 0; i < length; i++)
    {
        printf("#");
    }
    printf("\n");
}
